app/models/mention.rb

```diff
@@ -53,7 +53,7 @@ def self.reply_mentions
       if @lesson
         twitters = Twitter_API.new
         @client = twitters.client
-        to_send = mention.handler + ' ' + @lesson.intro + @lesson.thread_link
+        to_send = "#{mention.handler} #{@lesson.intro} #{@lesson.thread_link}"
```

Comment:
reply_mentions tweets the response to a mention, so it links to a threaded lesson but it tweets everytime the bot is called into action. I think that functionality makes more sense in the mention model.